### ISSUE TITLE
Update to OpenWrt BarrierBreaker 14.07

### ISSUE THREAD

**Issue by isantolin** [arduino/openwrt-yun#32]
It's a good idea update the core to OpenWrt BarrierBreaker 14.07

**Comment by ffissore** [comment]
Agree. It's in the todo list but it's not been planned yet. Would you like to contribute?

**Comment by hichris1234** [comment]
I'd be interested in helping with this, but I'm not quite sure where to start. Any hints you could give me?

**Comment by RedSnake64** [comment]
Here is my OpenWrt (BarrierBreaker 14.07) port for Arduino Yun:
https://github.com/RedSnake64/openwrt-yun

**Comment by isantolin** [comment]
@ffissore You can merge @RedSnake64 changes into main repository?

**Comment by ffissore** [comment]
Not yet. That kind of changes requires thorough testing because, at the end, users will rightfully ask us to support them. However we have already planned looking at @RedSnake64 work and that will happen soon. Thank you all for your heads up! It really makes our life simpler ;)

**Comment by q2dg** [comment]
Well...nowadays version is OpenWrt Chaos Calmer 15.05...

**Comment by RedSnake64** [comment]
@q2dg 
OpenWrt Chaos Calmer (15.05) for Arduino Yun is available here:
https://github.com/RedSnake64/openwrt-yun/releases

**Comment by q2dg** [comment]
@RedSnake64 Thanks a lot!!

@ffissore Could this be merged??

**Comment by q2dg** [comment]
Well, it seems Yún is pretty abandoned

**Comment by mbanzi** [comment]
Hi it's not abandoned. The notifications were not getting to the right people. We'll look into it shorlty

**Comment by isantolin** [comment]
@mbanzi Thanks Massimo, many of the changes just could be merged. We offer test changes and all the new stuff
<3

**Comment by szpeter80** [comment]
@mbanzi @isantolin @RedSnake64  First of all, thanks for your work guys! Any update on this? I'm using the Yun for my MT, and having official software from 2014 is not really an option... Embedded security is basically like a white unicorn, and just a quick search revealed Dropbear v2011.54 in the image is affected by CVE-2012-0920 But  I dont want to switch to Linino or Rpi (the Pi Zero hardware SPI and few GPIOs covers what i need) :(  Do you have a committed team/individual  for releasing newer software and comitted timeline?

**Comment by q2dg** [comment]
Well, surprise! New Yún shield comes with Chaos Calmer and its repository is https://github.com/arduino/openwrt-yun-1505
